src/test/java/redis/clients/jedis/csc/UnifiedJedisClientSideCacheTestBase.java

```diff
@@ -39,7 +40,8 @@ public void simple() {
       control.set("foo", "bar");
       assertEquals("bar", jedis.get("foo"));
       control.del("foo");
-      assertNull(jedis.get("foo"));
+      await().atMost(5, TimeUnit.SECONDS).pollInterval(50, TimeUnit.MILLISECONDS)
+          .until(() -> jedis.get("foo") == null);
```

Comment:
In Awaitility, it has `untilAsserted` method which is capable of doing assertions which is better than checking equality.